src/components/web-form-renderer.vue

```diff
@@ -386,6 +398,11 @@ html, body {
       "sendingDataModal": {
         "title": "Sending Submission",
         "body": "Your data is being submitted. Please don’t close this window until it’s finished."
+      },
+      "sessionTimeoutModal": {
+        "title": "Session expired",
+        "body": "Please log in {here} in a different browser tab and try again.",
+        "here": "here"
```

Comment:
Could you group these two messages in an object where the main message is named `full`? Doing so will automatically add comments for the two messages about the context in which "here" appears. You should see those comments show up in transifex/strings_en.json.

```json
"body": {
  "full": "Please log in {here} in a different browser tab and try again.",
  "here": "here"
}
```